rig-core/src/providers/openai/responses_api/streaming.rs

```diff
@@ -340,8 +340,8 @@ where
                 }
             }
 
-            // // Ensure event source is closed when stream ends
-            // event_source.close();
+            // Ensure event source is closed when stream ends
+            event_source.close();
```

Comment:
@joshua-mo-143 You commented this in #955, was that intentional?
I figured it would be safe to re-enable.